.github/workflows/generate.yml

```diff
@@ -33,8 +33,9 @@ jobs:
         scripts/gen-tests.sh
     
     - name: Git Commit/Push Changes
-      uses: actions-x/commit@v5
-      with:
-        branch: gen
-        files: k8s/
-        force: true
+      run: |
+        git add --force -N k8s/*/gen/*.yaml
+        git add -N k8s/
+        git commit -m "Automatically updated using GitHub Actions"
+        git push -f
```

Comment:
Should this be `push -f origin gen`? I don't think the upstream is set here.

Reply:
That's safer regardless, will change.